cloudflare/resource_cloudflare_account_member.go

```diff
@@ -83,6 +86,8 @@ func resourceCloudflareAccountMemberCreate(d *schema.ResourceData, meta interfac
 		return fmt.Errorf("failed to find ID in create response; resource was empty")
 	}
 
+	d.Set("status", r.Status)
```

Comment:
if an end user doesn’t set anything in the status, this is going to cause a permadiff as the result will be "pending" until they accept the invite. following that, once they accept, it will change to "accepted" resulting in a no-op apply to get the state in sync.